core_app/mls_integration.py

```
@@ -0,0 +1,314 @@
+        'count': len(_mls_data) if _mls_data else 0,
+        'last_updated': _mls_last_loaded,
+        'sample_mls_numbers': list(_mls_data.keys())[:5] if _mls_data else []
+    }
+
+def _safe_float(value) -> Optional[float]:
+    """Safely convert value to float"""
+    try:
+        if value is None or value == '':
+            return None
+        return float(str(value).replace(',', '').replace('$', ''))
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)